src/dawn_native/Device.cpp

```diff
@@ -178,8 +178,14 @@ namespace dawn_native {
     SwapChainBuilder* DeviceBase::CreateSwapChainBuilder() {
         return new SwapChainBuilder(this);
     }
-    TextureBuilder* DeviceBase::CreateTextureBuilder() {
-        return new TextureBuilder(this);
+    TextureBase* DeviceBase::CreateTexture(const TextureDescriptor* descriptor) {
+        TextureBase* result = nullptr;
+
+        if (ConsumedError(CreateTextureInternal(&result, descriptor)))
+        {
```

Comment:
nit: contrary to ANGLE we put braces at the end of the line. We have a clang format file, but it doesn't apply to all files yet (tests get really badly formatted because clang-format can't handle the "builder pattern"). Normally the formatting errors will be showed in the Travis build log with a diff you can apply to fix them. The only issue is that for now the LLVM apt repo is broken so the Travis build fails too early.